<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="INDP277LT" failures="1" tests="2" name="Test01" time="0.179" errors="0" timestamp="2021-01-09T02:51:52 IST">
  <testcase classname="listener.Demo" name="testMethod01" time="0.026">
    <failure type="java.lang.AssertionError" message="expected [3] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [3] but found [2]
at listener.Demo.testMethod01(Demo.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testMethod01 -->
  <testcase classname="listener.Demo" name="testMethod02" time="0.003"/>
</testsuite> <!-- Test01 -->
